### DISCUSSION TITLE
Is there a way to save a gds file just after global routing and before detailed routing?

### DISCUSSION THREAD

**Discussion by Leonardo-SA** [The-OpenROAD-Project/OpenROAD#5828] [Q&A]
Hi,

I would like to know how could I save a gds file while running OpenLane flow just after global routing step and before detailed routing step starts. The idea is to have a gds from global routing and the gds from signoff.

Thank you

**Comment by maliberty** [comment]
Questions about openlane should be directed to that project.  What would be the use of a gds from global routing as you would have no routes in your gds?
